litmus-portal/frontend/src/views/Analytics/ApplicationDashboards/Steps/TuneTheQueries/EditPanelsWizard/index.tsx

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable no-unused-expressions */
```

Comment:
remove eslint disables

Reply:
needed for the forEach loop

Reply:
This problem can be fixed, instead of using  `?` wrap the forEach loop inside an if condition.